src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1693,10 +1694,6 @@ namespace NServiceBus.Features
     {
         protected internal override void Setup(NServiceBus.Features.FeatureConfigurationContext context) { }
     }
-    public class BestPracticeEnforcement : NServiceBus.Features.Feature
```

Comment:
Needs a note in the upgrade guide and changes to http://docs.particular.net/nservicebus/messaging/best-practice-enforcement